tests/debugging/probe/test_remoteconfig.py

```diff
@@ -42,7 +42,9 @@ def remove_probes(self, *probe_ids):
                 pass
 
 
-def config_metadata(config_id=uuid4()):
+def config_metadata(config_id=None):
+    if config_id is None:
+        config_id = uuid4()
```

Comment:
is this a change in behavior?

a single global default uuid vs a new unique uuid every time this is called.

Reply:
I will ask @P403n1x87

Reply:
I don't think we care about this ID. If all tests pass then we are good 👍